src/FamilyAttributeData.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace SnowIO\AkeneoDataModel;
+
+class FamilyAttributeData
+{
+    public function getCode(): string
+    {
+        return $this->code;
+    }
+
+    public function isRequired(string $channel): bool
+    {
+        return $this->isRequired[$channel] ?? null;
```

Comment:
`null` does not satisfy the return type of `bool`. I would suggest a default value of `false` instead of `null`.